### PR TITLE
KLBP-100: Error pages.

### PR DESCRIPTION
**Pull request by iuliudascalu**

### CODE REVIEW

File: src/pages/templates/ErrorTemplate.tsx [hunk 1]
```diff
@@ -9,8 +9,15 @@ const ErrorTemplate = ({ children }: ITemplate): ReactElement => {
     <>
       <PageHeader isLogoutButtonVisible={false} />
       <ContainerFluid>
-        <div className="flex -mx-6 hFull h-full">
-          <div className="text-center">{children}</div>
+        <div className="-mx-6">
+          <div className="px-6 md:w-3/4 lg:w-2/4 mx-auto">
+            <Card
+              padding="py-20 px-12 md:px-28"
```
Comment (andreir-suciu) [inline src/pages/templates/ErrorTemplate.tsx:15]: I think we could add this classes under className prop and remove the padding here and in Card component as it is also added as a class.

Comment (mariuspop86) [reply src/pages/templates/ErrorTemplate.tsx:15]: For the moment we will use padding, size and what not, in className we will try to set only the classes that are extra. It's fine like this but we will need to come with a better solution next version.

**Review by emilomirpitechplus — APPROVED** [review]

**Review by andreir-suciu — APPROVED** [review]

File: src/pages/errors/Error500.tsx [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
 
 const Error500 = (): ReactElement => {
   return (
     <ErrorTemplate>
-      <div>
-        <h2>Something went wrong!</h2>
+      <div className="font-bold text-center">
+        <img src={error} alt="errorIcon" className="w-20 inline-block mb-10" />
+        <h2 className="text-4.5xl leading-tight">
+          Le site web ne parvient pas à
```
Comment (mariuspop86) [inline src/pages/errors/Error500.tsx:11]: Add translations

File: src/pages/errors/Error404.tsx [hunk 3]
```diff
@@ -1,11 +1,29 @@
...
 
 const Error404 = (): ReactElement => {
   return (
     <ErrorTemplate>
-      <div>
-        <h2>Page not found</h2>
+      <div className="font-bold text-center">
+        <img src={error} alt="errorIcon" className="w-20 inline-block mb-10" />
+        <h2 className="text-4.5xl leading-tight">
+          Attention
```
Comment (mariuspop86) [inline src/pages/errors/Error404.tsx:11]: Add translations.

File: src/pages/errors/Error401.tsx [hunk 4]
```diff
@@ -4,8 +4,14 @@ import ErrorTemplate from '../templates/ErrorTemplate'
 const Error401 = (): ReactElement => {
   return (
     <ErrorTemplate>
-      <div>
-        <h2>Unauthorized</h2>
+      <div className="font-bold">
+        <h2 className="text-4.5xl leading-tight">
+          Nous sommes désolés, mais cette page n&apos;est malheureusement pas
```
Comment (mariuspop86) [inline src/pages/errors/Error401.tsx:9]: Add translations.

**Review by mariuspop86 — APPROVED** [review]

File: src/assets/images/index.ts [hunk 5]
```diff
@@ -1,4 +1,5 @@
 import logo from './logo.svg'
 import spinner from './spinner.svg'
+import error from './error.svg'
 
-export { logo, spinner }
+export { logo, spinner, error }
```
Comment (ioan-iacob-netea) [inline src/assets/images/index.ts:5]: Please sort import/export elements.

**Review by ioan-iacob-netea — CHANGES_REQUESTED** [review]

File: src/pages/errors/Error500.tsx (hunk 2 above)
Comment (iuliudascalu) [reply src/pages/errors/Error500.tsx]: Done.

File: src/pages/errors/Error404.tsx (hunk 3 above)
Comment (iuliudascalu) [reply src/pages/errors/Error404.tsx]: Done.

File: src/pages/errors/Error401.tsx (hunk 4 above)
Comment (iuliudascalu) [reply src/pages/errors/Error401.tsx]: Done.

File: src/assets/images/index.ts (hunk 5 above)
Comment (iuliudascalu) [reply src/assets/images/index.ts]: Done.

File: src/lang/fr.json [hunk 6]
```diff
@@ -6,5 +6,14 @@
...
+  "homepage.welcome": "Bienvenue!",
+  "error.401.title": "Nous sommes désolés, mais cette page n'est malheureusement pas disponible.",
+  "error.401.description": "Merci de réessayer en utilisant le lien qui vous a été envoyé",
+  "error.404.title": "Attention",
+  "error.404.subtitle": "Une erreur est survenue !",
+  "error.404.description": "Veuillez nous excuser pour ce problème. Si cette erreur est bloquante ou persistante, veuillez noter le code et le statut de l'erreur et contactez votre administrateur au",
+  "error.404.link": "support@id-control.fr",
+  "error.500.title": "Le site web ne parvient pas à",
+  "error.500.subtitle": "afficher cette page.",
+  "error.500.description": "Il se peut que le site soit en cours de maintenance ou qu'une erreur de programmation se soit produite."
```
Comment (ioan-iacob-netea) [inline src/lang/fr.json:18]: Please order the translations.

**Review by ioan-iacob-netea — CHANGES_REQUESTED** [review]

File: src/lang/fr.json (hunk 6 above)
Comment (iuliudascalu) [reply src/lang/fr.json]: Done.

**Review by ioan-iacob-netea — APPROVED** [review]

**Pull request merged**